mindsdb_native/libs/phases/data_extractor/data_extractor.py

```diff
@@ -51,13 +51,14 @@ def _apply_sort_conditions_to_df(self, df):
         """
 
         # apply order by (group_by, order_by)
-        if self.transaction.lmd['model_is_time_series']:
-            asc_values = [order_tuple[ORDER_BY_KEYS.ASCENDING_VALUE] for order_tuple in self.transaction.lmd['model_order_by']]
```

Comment:
since `ORDER_BY_KEYS.ASCENDING_VALUE` and `ORDER_BY_KEYS.COLUMN` are not used, they may be removed completely

Reply:
good point, removed